### DISCUSSION TITLE
Map moveability

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#989] [General]
Hello
When selected, does map remain moveable all over the screen by dragging?
Thank you
![MapMoveable](https://user-images.githubusercontent.com/4082252/216251467-39cd27ab-506e-4563-8e0f-d0c299bbc6d7.jpg)

**Comment by wolfgang-ch** [comment]
You have detached the map view from the MT main window, this can be reverted when you drag the view tab (red box) somewhere into the MT main window

![image](https://user-images.githubusercontent.com/1283445/216277381-6f9d1cc3-082a-48a3-ab2e-c7290106d03f.png)
